.wharf-ci.yml

```diff
@@ -1,10 +1,10 @@
 build:
-  api:
+  wharf-api:
```

Comment:
Does this require any changes anywhere else?

Reply:
No